Source/utils/file_util.cpp

```diff
@@ -273,6 +278,7 @@ bool CreateDir(const char *path)
 
 void RecursivelyCreateDir(const char *path)
 {
+	if (strlen(path) == 0) return;

```

Comment:
Same regarding passing an empty path -- we shouldn't avoid calling this function with an empty path